docs/openj9_support.md

```diff
@@ -162,16 +162,13 @@ The project build and test OpenJDK with OpenJ9 on a number of platforms. The ope
 
 | Platform                                    | Operating system          |  Compiler                       |
 |---------------------------------------------|---------------------------|---------------------------------|
-| Linux x86 64-bit                            | Ubuntu 16.04              | gcc 4.8.5                       |
-| Linux on POWER&reg; LE 64-bit               | Ubuntu 16.04              | gcc 4.8.5                       |
-| Linux on IBM Z&reg; 64-bit                  | Ubuntu 16.04              | gcc 4.8.5                       |
```

Comment:
zLinux should be 7.4

Reply:
The OpenJ9 build machines have been updated, although the issue may not reflect this yet.

Reply:
- I've changed zlinux to 7.4. When I looked last week the issue was still open, hence updated to 7.3.

- cgroup changes are removed and will go into a separate PR once I get the go-ahead.

@pshipton  Are you ok to unblock for merging please?